cmd/kubeadm/app/cmd/phases/addons.go

```diff
@@ -64,7 +64,7 @@ func NewCmdAddon() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "addon",
 		Aliases: []string{"addons"},
-		Short:   "Installs required addons for passing Conformance tests",
+		Short:   "Installs required addons for passing conformance tests",
```

Comment:
Given the name throughout the code is [Conformance] the capitalization is consistent.